bin/utils/transform.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+import fsspec
+import pandas as pd
+
```

Comment:
Great to have a shared utils library, but it's a bit unconventional to have things other than programs in a bin directory. I would put this file into a lib directory, e.g. `lib/utils/transform.py` and then make sure `lib` is on `sys.path`.

Reply:
☝️ @kairstenfay (for reference on your wip branch)

Reply:
We discussed this on our calls, but wanted to record here for posterity: The (`bin/augur` dev wrapper](https://github.com/nextstrain/augur/blob/42a26edc229c32cd83e4bf272ce0bdada1c100c6/bin/augur) includes an example of import path manipulation. It's a bit verbose, and if desired, could be shortened into something like:

```python
import sys
from pathlib import Path
sys.path.insert(0, str(Path(__file__).parent.parent / "lib"))

from utils.transform import (…)
```